CIP-????/README.md

```diff
@@ -0,0 +1,209 @@
+---
+CIP: XXXX
+Title: A Standard for On-Chain Surveys and Polls
```

Comment:
```suggestion
Title: On-Chain Surveys and Polls
```
@Thomas-nada there's no reason to call a CIP a "Standard" which would be redundant at best in a title.  In reality it's not a standard until changing `Status:` from `Proposed` to `Active` (I'll trim this in the PR title as well).

Also @Ryun1 @Crypto2099 @perturbing (cc @gufmar @KtorZ & @thenic95) we should look at this title vs. [CIP-0094](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0094/README.md) and make sure they're sufficiently distinctive to tell them apart (I believe they are).